src/Compiler/SyntaxTree/SyntaxTrivia.fs

```diff
@@ -257,3 +257,10 @@ type SynMemberGetSetTrivia =
         AndKeyword: range option
         SetKeyword: range option
     }
+
+[<NoEquality; NoComparison>]
+type SynArgPatsNamePatPairsTrivia =
+    {
+        LeftParenRange: range
```

Comment:
Wouldn't a single range suffice?
Start position at left paren, end at the right one.

Reply:
It would and would be more consistent with `SynType.Paren` and `SynPat.Paren`, I'll update it.